quine-core/src/main/scala/com/thatdot/quine/graph/NodeActor.scala

```diff
@@ -198,20 +234,24 @@ private[graph] class NodeActor(
                 e
               }
             )
-        } else Future.successful(Done)
+        } else
+          Future.successful(Done)
```

Comment:
I believe CI will complain about this formatting as well, but the same `fixall` command I mentioned earlier should help you resolve that easily.